tests/cpp_validation_tests/test_validation.cpp

```diff
@@ -321,11 +384,13 @@ struct CaseParam {
     std::string calculation_method;
     std::string short_circuit_voltage_scaling;
     std::string tap_changing_strategy;
+    double err_tol = 1e-8;
+    Idx max_iter = 20;
+    Idx threading = -1;
     bool sym{};
```

Comment:
> I don't think so. If the tolerance is reached before 20 iterations, then it is; however, if the tolerance is not reached, the maximum number of iterations will be the safeguard (specially when creating new tests and they don't seem to converge).

@figueroa1395 are you sure?

Reply:
We were talking about different types of redundancy. Thanks for pointing this out. Changes will show in following commit. Resolving.